src/main/java/org/opentripplanner/model/branding/Branding.java

```diff
@@ -0,0 +1,79 @@
+package org.opentripplanner.model.branding;
+
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.model.TransitEntity;
+
+public class Branding extends TransitEntity {
```

Comment:
All `opentripplanner.model` classes should have JavaDoc. Important to mention is GTFS and NeTEx support.